src/json-formatter.js

```diff
@@ -31,13 +31,55 @@ angular.module('jsonFormatter', ['RecursionHelper'])
     return typeof object;
   }
 
+  function isObject(object) {
+    return object && typeof object === 'object';
+  }
+
+  function isArray(object) {
+    return Array.isArray(object)
+  }
+
+  function parseValue (object, value) {
+    var type = getType(object);
+    if (type === 'null') {
+      return 'null';
+    }
+    if (type === 'undefined') {
+      return 'undefined';
+    }
+    if (type === 'string') {
+      value = '"' + escapeString(value) + '"';
+    }
+    if (type === 'function'){
+
+      // Remove content of the function
+      return object.toString()
```

Comment:
Is this different than our other function-removing-regex? If no, then let's put this logic in a function and reuse it here.